features/page_objects/external_user_manage_users_page.rb

```diff
@@ -1,3 +1,3 @@
-class ManageUsersPage < BasePage
+class ExternalUserManageUsersPage < BasePage
```

Comment:
Happy for this to change but I do not see how it is related to the PR. Likewise for the changes to `features/step_definitions/manage_users_steps.rb` and the change to line 10 of `features/support/hooks.rb`. Perhaps it should be split into a separate commit.